utils/data-processing-family-task-submitter-container/pom.xml

```diff
@@ -0,0 +1,162 @@
+<!--
+
+    Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.github.cafdataprocessing</groupId>
+        <artifactId>processing-utilities-aggregator</artifactId>
+        <version>1.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>data-processing-family-task-submitter-container</artifactId>
+    <packaging>pom</packaging>
+
+    <properties>
+        <cafFamilyTaskSubmitterContainerName>
+            ${dockerDataProcessingOrg}data-processing-family-task-submitter${dockerProjectVersion}
+        </cafFamilyTaskSubmitterContainerName>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.cafdataprocessing</groupId>
+            <artifactId>data-processing-family-task-submitter</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.github.krallin</groupId>
+            <artifactId>tini</artifactId>
+            <scope>runtime</scope>
+            <type>exe</type>
+        </dependency>
```

Comment:
We don't need to pull in `tini` any more as we included it in the custom java base image.

Reply:
OK, removed.